docgen/src/gettingstarted.md

```diff
@@ -6,49 +6,323 @@ nav_groups:
 nav_sort: 0
 ---
 
-This is the home of `react-instantsearch`, a set of [React](https://facebook.github.io/react/) components to create unique instant-search experiences using [Algolia](https://www.algolia.com/) search engine.
+## Welcome to react-instantsearch
 
-**This is an alpha release, everything may be broken**.
+React-instantsearch is the ultimate toolbox for creating instant search
+experience using React.js and Algolia.
 
-Links:
-- [Chat with us](https://gitter.im/algolia/instantsearch.js) on gitter, we are active and waiting for feedback
-- Code is at <https://github.com/algolia/instantsearch.js/tree/v2>, jump in
+In this tutorial, you'll learn how to:
 
+ - add react-instantsearch in your react project
+ - bootstrap react-instantsearch
+ - display results from Algolia
+ - add components to filter the results
+ - connect your own component to the search
 
-## Example
+## Before we start
+
+As mentionned earlier react-instantsearch is meant to be used with Algolia.
+
+Therefore, you'll need the credentials to an Algolia index. There are two
```

Comment:
Since this getting started is also a getting started tutorial, I would just not leave the choice to the user and force him to use the demo credentials.

While still insisting that he can also create his own account later on.

The goal being, in the getting started, I should not have to make any choices, I just want to try the thing, not to take decisions (at this point)